### ISSUE TITLE
Error Build compileReleaseJavaWithJavac. Please Help

### ISSUE THREAD

**Issue by MackayBravo** [wumke/react-native-local-notifications#4]
Virsions
```
    "react-native": "^0.47.1",
    "react-native-local-notifications": "^0.1.0",
```
Error
```
:react-native-local-notifications:compileReleaseJavaWithJavac
:react-native-local-notifications:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
C:\web\apps\reactnative\demonotify\node_modules\react-native-local-notifications\android\src\main\java\com\github\wumke\RNLocalNotifications\RNLocalNotificationsPackage.java:26: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-local-notifications:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-local-notifications:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

**Comment by cravi24** [comment]
react-native removed createJSModules in version 0.47.1 which caused this issue. You may want to revert back to react-native@0.46.4 till a fix is pushed

**Comment by wumke** [comment]
I will fix this and create a new major version and put the react native versions in the readme as soon as I find the time. Thanks for reporting! I never use the latest version of react native since that might break other packages I use to :-)

**Comment by wumke** [comment]
fixed in 1.0.0
